core/src/main/java/com/sequenceiq/cloudbreak/structuredevent/converter/RdsConfigToRdsDetailsConverter.java

```diff
@@ -23,6 +24,11 @@ public RdsDetails convert(RDSConfig source) {
         rdsDetails.setConnectorJarUrl(source.getConnectorJarUrl());
         rdsDetails.setCreationDate(source.getCreationDate());
         rdsDetails.setDatabaseEngine(source.getDatabaseEngine().name());
+        if (DatabaseVendor.EMBEDDED == source.getDatabaseEngine()) {
```

Comment:
next time you can use a simpler version of this:
`rdsDetails.setExternal(DatabaseVendor.EMBEDDED != source.getDatabaseEngine())`
if you want you can extract into a field as well.